deploy/examples/r.yaml

```diff
@@ -0,0 +1,418 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
this file will removed later as it added in this commit for my local testing purpose only